cloudinary.js

```
@@ -0,0 +1,19 @@
+const cloudinary = require('cloudinary').v2
+const Multer = require('multer')
+
+cloudinary.config({
+  cloud_name: process.env.CLOUD_NAME,
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)